source/Generic/GameRelations/GameRelationsSettingsView.xaml

```diff
@@ -221,7 +221,40 @@
                                 </DockPanel>
                             </Grid>
                         </Expander>
-                        
+
+                        <Expander Margin="0,10,0,0" Header="Advanced">
+                            <StackPanel>
+                                <StackPanel Orientation="Horizontal" Margin="0,10,0,0">
+                                    <TextBlock VerticalAlignment="Center">Jacard similarity requirement per field:</TextBlock>
```

Comment:
Missing translation string

Reply:
I think the string should remain only as "Minimum similarity to match games" with a more detailed explanation in a tooltip, including saying that it uses Jacard.